docs/source/jupyter.rst

```diff
@@ -19,3 +19,4 @@ Jupyter notebooks
    /examples/Visualizing the SpaceX Tesla Roadster trip to Mars.ipynb
    /examples/Natural and artificial perturbations.ipynb
    /examples/Porkchops with poliastro.ipynb
+   /examples/Multirevolutions solution in Lambert's problem.ipynb
```

Comment:
You will have to rename the file so it doesn't have apostrophe! It used to bring problems in the past